### PR TITLE
NF: undo message is more specific

### PR DESCRIPTION
**Pull request by Arthur-Milchior**
Currently the message state that it undoes "(un)mark card" and "(un)suspend cards"

It is now really easy to correct this and put a more specific message. 
I expect it to make https://github.com/ankidroid/Anki-Android/issues/7162 easier to debug

2020-09-19-220932_622x1151_scrot
2020-09-19-220743_622x1151_scrot
2020-09-19-220739_622x1151_scrot
2020-09-19-220723_622x1151_scrot

### CODE REVIEW

File: AnkiDroid/src/main/res/values/01-core.xml [hunk 1]
```diff
@@ -78,7 +78,9 @@
     <string name="undo_action_bury_card">bury card</string>
     <string name="undo_action_bury_note">bury note</string>
     <string name="undo_action_suspend_card">suspend card</string>
+    <string name="undo_action_unsuspend_card">suspend card</string>
     <string name="undo_action_suspend_note">suspend note</string>
+    <string name="undo_action_unsuspend_note">suspend note</string>
```
Comment (david-allison-1) [inline AnkiDroid/src/main/res/values/01-core.xml:83]:
```suggestion
    <string name="undo_action_unsuspend_note">unsuspend note</string>
```

File: AnkiDroid/src/main/res/values/01-core.xml [hunk 2]
```diff
@@ -78,7 +78,9 @@
     <string name="undo_action_bury_card">bury card</string>
     <string name="undo_action_bury_note">bury note</string>
     <string name="undo_action_suspend_card">suspend card</string>
+    <string name="undo_action_unsuspend_card">suspend card</string>
```
Comment (david-allison-1) [inline AnkiDroid/src/main/res/values/01-core.xml:81]:
```suggestion
    <string name="undo_action_unsuspend_card">unsuspend card</string>
```

File: AnkiDroid/src/main/java/com/ichi2/async/CollectionTask.java [hunk 3]
```diff
@@ -884,8 +884,8 @@ public UndoChangeDeckMulti(Card[] cards, long[] originalDids) {
         private final List<Note> originalUnmarked;
 
 
-        public UndoMarkNoteMulti(List<Note> originalMarked, List<Note> originalUnmarked) {
-            super(Collection.DismissType.MARK_NOTE_MULTI);
+        public UndoMarkNoteMulti(List<Note> originalMarked, List<Note> originalUnmarked, boolean hasUnmarked) {
```
Comment (david-allison-1) [inline AnkiDroid/src/main/java/com/ichi2/async/CollectionTask.java:887]: Could you document what `hasUnmarked` means?

File: AnkiDroid/src/main/java/com/ichi2/async/CollectionTask.java [hunk 4]
```diff
@@ -780,9 +780,9 @@ private TaskData doInBackgroundDismissNote(TaskData param) {
         private final Card[] cards;
         private final boolean[] originalSuspended;
 
-
-        public UndoSuspendCardMulti(Card[] cards, boolean[] originalSuspended) {
-            super(Collection.DismissType.SUSPEND_CARD_MULTI);
+        public UndoSuspendCardMulti(Card[] cards, boolean[] originalSuspended,
```
Comment (david-allison-1) [inline AnkiDroid/src/main/java/com/ichi2/async/CollectionTask.java:783]: ditto - quick comment on meaning of `hasUnsuspended`

**Review by david-allison-1 — CHANGES_REQUESTED** [review]

File: AnkiDroid/src/main/res/values/01-core.xml (hunk 1 above)
Comment (Arthur-Milchior) [reply AnkiDroid/src/main/res/values/01-core.xml]: Deleted. I reuse strings and thought I already deleted the strings I temporarily added

File: AnkiDroid/src/main/res/values/01-core.xml (hunk 2 above)
Comment (Arthur-Milchior) [reply AnkiDroid/src/main/res/values/01-core.xml]: same

File: AnkiDroid/src/main/java/com/ichi2/async/CollectionTask.java (hunk 3 above)
Comment (Arthur-Milchior) [reply AnkiDroid/src/main/java/com/ichi2/async/CollectionTask.java:887]: Done

File: AnkiDroid/src/main/java/com/ichi2/async/CollectionTask.java (hunk 4 above)
Comment (Arthur-Milchior) [reply AnkiDroid/src/main/java/com/ichi2/async/CollectionTask.java:783]: Done

I kept the same name for the argument and for the parameter

File: AnkiDroid/src/main/java/com/ichi2/async/CollectionTask.java (hunk 3 above)
Comment (david-allison-1) [reply AnkiDroid/src/main/java/com/ichi2/async/CollectionTask.java:887]: Pushed? I don't see it

**Review by david-allison-1 — APPROVED** [review]
One request for documentation that doesn't seem to have gone in yet - looks good other than that.

**Comment by Arthur-Milchior** [comment]
Oops. 
> failed to push some refs to 'git@github.com:Arthur-Milchior/Anki-Android.git'  [Type `$' for details]

I retried

**Review by mikehardy — APPROVED** [review]
LGTM thanks!

**Pull request merged**